db_test.go

```diff
@@ -30,15 +30,15 @@ const pageHeaderSize = 16
 
 // meta represents a simplified version of a database meta page for testing.
 type meta struct {
-	magic    uint32
```

Comment:
This change is not related to the advertised subject of the PR.

Reply:
It's required to make golangci-lint pass.